rpm/generic/zfs-dkms.spec.in

```diff
@@ -36,7 +36,7 @@ Obsoletes:      spl-dkms
 Provides:       %{module}-kmod = %{version}
 AutoReqProv:    no
 
-%if 0%{?rhel}%{?fedora}%{?suse_version}
+%if (0%{?fedora}%{?suse_version}) || (0 < 0%{?rhel} && 0%{?rhel} < 9)
```

Comment:
This RHEL clause makes no sense. It should be `(0%{?rhel} && 0%{?rhel} < 9)` or `(0%{?rhel} > 0 && 0%{?rhel} < 9)`. I'm not even sure what a `< 0` does here?

Reply:
My vague recollection is that some older specfile version didn't allow you to get away with just `0%{?rhel}`.

Or maybe my brain is rotten. It's a toss-up, really. :)